driver-async/src/main/com/mongodb/async/client/MongoClient.java

```diff
@@ -39,13 +39,15 @@
     /**
      * Creates a client session.
      *
+     * <p>Note: ClientSessions cannot be used concurrently, only one can be used at a time.</p>
```

Comment:
This isn't quite right.  More than one can be used at a time, but one can't be used concurrently in multiple operations.  How about:

     A ClientSession instance can not be used concurrently in multiple asynchronous operations.

Reply:
Done.